Lickability/PinpointKit
# Swift 2.3 Routine `develop` Merge

:memo: This PR is pointed at the `swift-2.3` branch

## What It Does

- Merges `develop`
- Runs `bundle exec pod install` on `/Example` to fix conflicts in the Pods project

## How to Test

- Check out this branch
- Build and run
- Smoke test Pinpoint features

## Notes

This merge was relatively painless due to the limited changes made by the 2.3 migrator. I expect the corresponding 3.0 one to require a lot more conflict resolution.
